content/english/phaser-space-invaders-game/1.starting-up.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Starting Up"
```

Comment:
For all the files, please remove the #. prefix in the names - we will just use the weights within the files for ordering.